website/spec/Connections.md

```diff
@@ -1,9 +1,14 @@
-Relay Cursor Connections Specification
+GraphQL Cursor Connections Specification
 --------------------------------------
 
-Relay's support for pagination relies on the GraphQL server exposing
-connections in a standardized way. In the query, the connection model
-provides a standard mechanism for slicing and paginating the result set.
+This specification aims to provide options for GraphQL clients to consistently 
+handle pagination with support for related metadata via a GraphQL server. This
+spec proposes calling them connections and treats pageable list in a standardized 
+way. 
+
+In the query, the connection model provides a standard mechanism for slicing 
+and paginating the result set.
```

Comment:
I have a hard time understanding this sentence. I know you didn’t write this, but do you understand what it’s trying to convey?

Reply:
I think it's talking about always using the same args in each connection for pagination

Reply:
e.g. `before` `after` `next` `the_other_one`

Reply:
After reading further, I think they mean in the literal query _text_, and then what you said yeah